test/domain/model/contact_test.dart

```diff
@@ -3,64 +3,111 @@ import 'package:fluffychat/domain/model/contact/contact.dart';
 import 'package:fluffychat/domain/model/contact/hash_details_response.dart';
 
 void main() {
-  group('calLookupAddress', () {
-    group('calLookupAddress sha256', () {
+  group('calLookupAddress for email', () {
+    const contactWithEmail = Contact(
+      email: 'alice@example.com',
+    );
+    test('calLookupAddress no algorithm', () {
       const hashDetails = HashDetailsResponse(
-        algorithms: {'sha256'},
         lookupPepper: 'matrixrocks',
       );
-      test('calLookupAddress sha256 example 1', () {
-        const contactWithEmail = Contact(
-          email: 'alice@example.com',
-        );
-        expect(
-          contactWithEmail.calLookupAddress(hashDetails: hashDetails),
-          '4kenr7N9drpCJ4AfalmlGQVsOn3o2RHjkADUpXJWZUc',
-        );
-      });
-
-      test('calLookupAddress sha256 example 2', () {
-        const contactWithEmail = Contact(
-          email: 'bob@example.com',
-        );
-        expect(
-          contactWithEmail.calLookupAddress(hashDetails: hashDetails),
-          'LJwSazmv46n0hlMlsb_iYxI0_HXEqy_yj6Jm636cdT8',
-        );
-      });
+      expect(
+        contactWithEmail.calLookupAddress(hashDetails: hashDetails),
+        '4kenr7N9drpCJ4AfalmlGQVsOn3o2RHjkADUpXJWZUc',
+      );
+    });
 
-      test('calLookupAddress sha256 phone', () {
-        const contactWithPhone = Contact(
-          phoneNumber: '18005552067',
-        );
-        expect(
-          contactWithPhone.calLookupAddress(hashDetails: hashDetails),
-          'nlo35_T5fzSGZzJApqu8lgIudJvmOQtDaHtr-I4rU7I',
-        );
-      });
+    test('calLookupAddress no pepper', () {
+      const hashDetails = HashDetailsResponse(
+        algorithms: {'sha256'},
+      );
+      expect(
+        contactWithEmail.calLookupAddress(hashDetails: hashDetails),
+        'VcXk6XdYWm8SS3XJlWr9fg_RA347SJPO7zkenhzr4wY',
+      );
     });
-  });
 
-  group('calLookupAddress none algorithm', () {
-    const hashDetails = HashDetailsResponse(algorithms: {'none'});
-    test('calLookupAddress none algorithm email', () {
-      const contactWithEmail = Contact(
-        email: 'alice@example.com',
+    test('calLookupAddress algorithm was not supported', () {
+      const hashDetails = HashDetailsResponse(
+        algorithms: {'sha1'},
+        lookupPepper: 'matrixrocks',
       );
       expect(
         contactWithEmail.calLookupAddress(hashDetails: hashDetails),
         'alice@example.com email',
       );
     });
 
-    test('calLookupAddress none algorithm phone', () {
+    test('calLookupAddress sha256 email alice@example.com', () {
+      const hashDetails = HashDetailsResponse(
+        algorithms: {'sha256'},
+        lookupPepper: 'matrixrocks',
+      );
+      expect(
+        contactWithEmail.calLookupAddress(hashDetails: hashDetails),
+        '4kenr7N9drpCJ4AfalmlGQVsOn3o2RHjkADUpXJWZUc',
+      );
+    });
+
+    test('calLookupAddress sha256 email bob@example.com', () {
```

Comment:
what is the different from the case : `test('calLookupAddress sha256 email alice@example.com',`

Reply:
it is example from spec, so I want to test it